docs/_running-tests/chrome.md

```diff
@@ -13,8 +13,7 @@ run Chrome Dev on Linux:
 
 Extra arguments to Chrome can be passed with `--binary-args`.
 
-To enable all [experimental web platform features](https://www.chromium.org/blink/runtime-enabled-features)
-[Experimental Web Platform features](chrome://flags/#enable-experimental-web-platform-features):
+To enable all [experimental web platform features](https://www.chromium.org/blink/runtime-enabled-features) (chrome://flags/#enable-experimental-web-platform-features):
```

Comment:
This seems wrong per the commit message; I presume you meant to keep the https link here.

Reply:
I did keep one https link, it's just chrome:// which seems impossible to make a link from.